docs/examples/dl3_tool_config.json

```diff
@@ -12,7 +12,14 @@
   },
   "DL3FixedCuts": {
     "fixed_gh_cut": 0.7,
+    "fixed_gh_efficiency": 0.9,
     "fixed_theta_cut": 0.2,
+    "fixed_theta_containment": 0.68,
     "allowed_tels": [1]
+  },
+  "DataBinning": {
+    "reco_energy_min": 0.01,
+    "reco_energy_max": 100,
+    "reco_energy_n_bins_per_decade": 5.5
```

Comment:
Why 5.5?

Reply:
This was arbitrary choice made long time ago. I will change it to 5 as per the CTA standard.